spec/rubocop/cop/lint/void_spec.rb

```diff
@@ -254,4 +254,13 @@ def foo=(rhs)
       nil
     RUBY
   end
+
+  it 'handles method with irange block' do
+    expect_no_offenses(<<~RUBY)
+      def foo
+        1..100.times.each { puts 1 }
+        raise
```

Comment:
That's just my two cents. `do_something` may be better as an example than `raise`.

```suggestion
        do_something
```